src/dosbox.cpp

```diff
@@ -624,20 +619,43 @@ void DOSBOX_Init(void) {
 	};
 	pstring->Set_values(midi_devices);
 	pstring->Set_help(
-	        "Device that will receive the MIDI data from emulated MPU-401.\n"
+	        "Device that will receive the MIDI data (from the emulated MIDI\n"
+	        "interface - MPU-401).\n"
 #if C_FLUIDSYNTH
 	        "Use 'fluidsynth' to select built-in software synthesiser,\n"
 	        "see the fluidsynth section for detailed configuration.\n"
 #endif
-	        "Use 'auto' to pick the first working device.");
-
-	Pstring = secprop->Add_string("midiconfig",Property::Changeable::WhenIdle,"");
-	Pstring->Set_help("Special configuration options for the device driver. This is usually the id or part of the name of the device you want to use\n"
-	                  "(find the id/name with mixer/listmidi).\n"
-	                  "Or in the case of coreaudio, you can specify a soundfont here.\n"
-	                  "When using a Roland MT-32 rev. 0 as midi output device, some games may require a delay in order to prevent 'buffer overflow' issues.\n"
-	                  "In that case, add 'delaysysex', for example: midiconfig=2 delaysysex\n"
-	                  "See the README/Manual for more details.");
+	        "Use 'auto' to pick the first working external device.");
+
+	pstring = secprop->Add_string("midiconfig", when_idle, "");
+	pstring->Set_help(
+	        "Configuration options for the selected 'mididevice'.\n"
```

Comment:
The wording is somewhat circular: _Depending on the midi device, it selects the midi device_.

Here's my understanding (and possible wording.. albeit a bit long):

Dosbox can send MIDI sequences through a selected **interface** (ALSA, CoreAudio, and WindowsMIDI/win32), which may play the MIDI itself in the case of CoreAudio (when a path to an SF2 is provided), or route the sequence to one or more players (physical mt32, Qsynth, Timidity, munt, ...) that are connected to the interface, via a **connection identifier**.

We want the user to choose their _MIDI interface_ and either _connection identifier_ or _SF2_ in the case of CoreAudio.

Ideally we could rename these settings.. but that would hurt backward compatibility. Maybe we can clarify the descriptions.

Reply:
Thanks for the updates, @dreamer!

Will take another try tomorrow at this midiconfig text with fresh eyes. Don't want us burning too much time on it.

Reply:
Yes, it *is* too long. Every sentence added means someone will not read it at all. Also "connection identifier" would be inventing yet another, new term: on Linux it's: MIDI sequencer port (but I have no idea about nomenclature on other OSes).

Let's keep this as brief as possible

How about:

    midiconfig: Configuration options for the selected MIDI interface.
                This is usually the id or name of the MIDI synthesiser you want
                to use (find the id/name with DOS command 'mixer /listmidi').

Reply:
You nailed it with "Configuration options for the selected MIDI interface."! Thanks @dreamer.